akka-stream/src/main/boilerplate/akka/japi/tuple/Tuple.scala.template

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.japi.tuple
```

Comment:
True, so we'll have to:

* copy these to `release-2.3` and `master`
* I'm not convinced about `akka.japi.tuple`, let's keep them in `akka.japi`
* once 2.3.12 is released, bump streams dependency and remove them from streams (2.3-dev)

I created a ticket for the moving things part: https://github.com/akka/akka/issues/17719